aip_xx1_launch/launch/lidar.launch.xml

```diff
@@ -24,6 +25,12 @@
         <arg name="vehicle_mirror_param_file" value="$(var vehicle_mirror_param_file)"/>
         <arg name="use_pointcloud_container" value="$(var use_pointcloud_container)"/>
         <arg name="container_name" value="$(var pointcloud_container_name)"/>
+        <arg name="angle_range" value="[0.0, 360.0]"/>
+        <arg name="vertical_bins" value="128"/>
+        <arg name="horizontal_ring_id" value="64"/>
```

Comment:
[Q]
Is `ring_id: 64` correct?
Has this algorithm taken into account the arrangement pattern of the lasers?
![image](https://github.com/tier4/aip_launcher/assets/37187849/6a6b5758-d5a5-45be-998e-941b71da4e6e)

Reply:
@miursh 
As confirmation in Rviz, the velodyne ring IDs are arranged in the order as the assumption of the current algorithm that ring_id will be whether Top to Bottom or Bottom to Top. 
In VLS128 the order is from the bottom.
![image](https://github.com/tier4/aip_launcher/assets/94814556/398ba3bf-6e3d-4edd-8998-89a9e3e5eb18)

Reply:
@badai-nguyen 
Ah, I see. Understood. 
However, is 64 is really horizontal? 
Typically, there are more downward-facing lasers than upward-facing ones, so I doubt that the center would be horizontal.

Reply:
@miursh Thank you for your comment. 
Currently, I checked on the kind of empty environment and I think 64 is acceptable.  
![image](https://github.com/tier4/aip_launcher/assets/94814556/44f08131-a953-4a51-bdf0-b30a1cdd59f0)
![image](https://github.com/tier4/aip_launcher/assets/94814556/c0d8f9c3-f944-443e-9148-7416af0e2808)

I think it also could be adjusted if any feedback after the function is enabled for operation.